format/spec.md

```diff
@@ -1208,6 +1224,7 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
 | **`struct`**       | `group`                                                            |                                             |                                                                |
 | **`list`**         | `3-level list`                                                     | `LIST`                                      | See Parquet docs for 3-level representation.                   |
 | **`map`**          | `3-level map`                                                      | `MAP`                                       | See Parquet docs for 3-level representation.                   |
+| **`variant`**      | `group` with `metadata` and `value` fields. `metadata` and `value` must not be assigned field IDs.| `VARIANT`                                   | See Parquet docs for Variant encoding and Variant shredding encoding. |
```

Comment:
If these don't have field ID, how should the reader locate the contents of a variant iceberg field?
are these mapped by name?

maybe that's obvious, but let's make it explicit.